src/coreclr/utilcode/stresslog.cpp

```diff
@@ -748,7 +748,14 @@ FORCEINLINE void ThreadStressLog::LogMsg(unsigned facility, int cArgs, const cha
     if (offs >= StressMsg::maxOffset)
     {
 #ifdef _DEBUG
+        // ASAN-TODO: An ASAN-enabled CoreCLR is too large and it's size exceeds Stress:Msg::maxOffset
+        // Don't force a debugger breakpoint here, otherwise we'll hit this every time we run
+        // tests on an ASAN-based CoreCLR.
+        // Consider revving the StressLog format (and SOS) versions to make the StressLog more reliable for larger
+        // module cases.
```

Comment:
Do we want to rev the stresslog format to ensure we are sufficiently guarded against the format strings being out of range? Is this a scenario we can hit in production with the single-file-host as well? I think so, but I'm not 100% positive.

Reply:
This will be addressed with https://github.com/dotnet/runtime/pull/83855